apps/graphql/src/definitions/Owner.js

```diff
@@ -151,6 +153,27 @@ export const resolvers = {
       const hasWritePermission = owner.$checkWritePermission(context.user);
       return hasWritePermission ? ["read", "write"] : ["read"];
     },
+    async consumptionRatio(owner) {
+      const consumedScreenshots = await Screenshot.query()
+        .joinRelated("screenshotBucket.repository")
+        .where({ [`screenshotBucket:repository.${owner.type()}Id`]: owner.id })
+        .where("screenshotBucket:repository.private", "true")
+        .where(
+          "screenshots.createdAt",
+          ">=",
+          moment().startOf("month").toISOString()
+        )
+        .resultSize();
+      const purchases = await getActivePurchases(owner);
+      if (purchases.length === 0) {
+        return 0;
```

Comment:
If there is no purchase, we fallback to the free plan. That is currently unlimited, but later not. So no "0" in the code, but read it from database.